components/SpotlightMenu/SpotlightMenu.tsx

```diff
@@ -1,5 +1,26 @@
 import React, { FunctionComponent } from 'react';
 
-const SpotlightMenu: FunctionComponent = ({ children }) => <div>{children}</div>;
+interface SpotlightMenuProps {
+  show?: boolean;
```

Comment:
Handle this outside the SpotlightMenu component.

Reply:
Also handled in a parent component